### DISCUSSION TITLE
debbuild support?

### DISCUSSION THREAD

**Discussion by brianjmurrell** [rpm-software-management/mock#1453] [General]
I've just come across https://github.com/debbuild/debbuild/.  I wonder what the feasibility of using it to support building of DEB packages in mock are.

Any thoughts?

**Comment by Conan-Kudo** [comment]
It's somewhat feasible. I did [a very simple prototype of the phases in GitLab CI](https://gitlab.com/Conan_Kudo/demo-multidistro-pipeline/-/tree/main/.gitlab-ci). It would definitely be possible to transpose this into something that mock can do. Also, nowadays `apt` can be massaged into bootstrapping an operating system chroot (kiwi and mkosi both do this), which fits the model mock wants.

**Comment by xsuchy** [comment]
It is technically doable. But you need to have answer for:
 * why? Do we have a group of users that would love to have. Does it significantly improve their workflow?
 * who will do that? Currently most of the contribution to Mock comes from people who use rpm-based distro. E.g. I, myself, has zero motivation to work on this. To implement and maintain this you would need somebody fully focused and dedicated for this.
 * how? Hmm, skip this one until we have answers to the first two questions.
